integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionTableLoadingTestCase.scala

```diff
@@ -640,6 +653,10 @@ class StandardPartitionTableLoadingTestCase extends QueryTest with BeforeAndAfte
     }
   }
 
+  override def afterEach(): Unit = {
+    CarbonProperties.getInstance()
```

Comment:
no need for afterEach, in the same testcase where you are setting. you can reset

Reply:
Actually it is a good practice to do cleanup/reset in afterEach so that even if testcase fails abnormally with some exceptions, it gets cleaned/reset in afterEach. And any testcases((if exists) below it will not be affected  with previous testcase failures.

Reply:
If test case fails, we need to fix the test case. 
So, just because test case can fail. we should not add before each as it is redundant for other test case and it will increase CI time.

Reply:
ok. Removed this afterEach.